.github/workflows/pr.yaml

```diff
@@ -103,6 +113,8 @@ jobs:
           VITE_SLACK_TEAM_ID: ${{ secrets.SLACK_TEAM_ID_TEST }}
           VITE_STAGE: ${{ env.STAGE }}
           VITE_CI: true
+          VITE_DB_HOST: ${{ steps.sst-output.outputs.host }}
           VITE_DB_NAME: ${{ steps.sst-output.outputs.database }}
-          VITE_DB_SECRET_ARN: ${{ steps.sst-output.outputs.secretArn }}
-          VITE_DB_CLUSTER_ARN: ${{ steps.sst-output.outputs.clusterArn }}
+          VITE_DB_USER: ${{ steps.sst-output.outputs.dbUser }}
+          VITE_DB_PASSWORD: ${{ steps.sst-output.outputs.dbPassword }}
+          VITE_DB_FUNCTION_NAME: ${{ steps.sst-output.outputs.dbFunction }}
```

Comment:
Why do we need this?

Reply:
You mean the funtion name? We are invoking the function by the name, from the tests.

Reply:
Dont want no cut out the tests, just remove the run, temorary